data/src/main/java/io/gnosis/data/models/Chain.kt

```diff
@@ -0,0 +1,24 @@
+package io.gnosis.data.models
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import io.gnosis.data.models.Chain.Companion.TABLE_NAME
+
+@Entity(tableName = TABLE_NAME)
+data class Chain(
```

Comment:
we could combine Chain & ChainInfo into one class as we currently do not have separate dto's and use models from cgw directly

Reply:
How would we handle embedded types like NativeCurrency and ChainTheme? Flatten them into the chain table or have relations?

Reply:
flatten them